src/edu/washington/escience/myria/util/JsonAPIUtils.java

```diff
@@ -179,4 +181,24 @@ public static HttpURLConnection ingestData(final String masterHostname, final in
     conn.getResponseCode();
     return conn;
   }
+
+  /**
+   * @param masterHostname master hostname
+   * @param apiPort rest api port
+   * @return a HTTPURLConnection instance of retrieving responses.
+   * @throws IOException if IO errors
+   * */
+  public HttpURLConnection getDatasetStatus(final String masterHostname, final int apiPort, final RelationKey key)
```

Comment:
Where is this new method used? I see the name used elsewhere but not the method defined in this class.